Sources/Tools/MaxConvert/hsMaterialConverter.cpp

```diff
@@ -4786,43 +4796,43 @@ static int ICompareDoneMats(const hsMaterialConverter::DoneMaterialData* one,
         plLightMapComponent* twoLM = twoNode->GetLightMapComponent();
         if( oneLM != twoLM )
         {
-            return oneNode > twoNode ? 1 : -1;
+            return oneNode > twoNode ? false : true;
```

Comment:
```suggestion
            return oneNode < twoNode;
```
(Not `<=`, since the compare operator in `std::sort` expects `true` only if the first object is less than the second...  An equal comparison would be one where the function returns false for both directions).